ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/NVidia_Cuda_Toolkit/tasks/main.yml

```diff
@@ -37,27 +37,6 @@
     - ansible_architecture == "ppc64le"
   tags: nvidia_cuda_toolkit
 
-- name: Download DKMS - RHEL 7 PPC64LE
-  get_url:
-    url: https://dl.fedoraproject.org/pub/epel/7/ppc64le/Packages/d/dkms-2.4.0-1.20170926git959bd74.el7.noarch.rpm
-    dest: /tmp/dkms.noarch.rpm
-  when:
-    - cuda_installed.stat.isdir is not defined
-    - ansible_distribution == "RedHat"
-    - ansible_distribution_major_version == "7"
-    - ansible_architecture == "ppc64le"
-  tags: nvidia_cuda_toolkit
-
-- name: Install DKMS RPM - RHEL 7 PPC64LE
-  yum: name=/tmp/dkms.noarch.rpm state=installed
-  ignore_errors: True
-  when:
-    - cuda_installed.stat.isdir is not defined
-    - ansible_distribution == "RedHat"
-    - ansible_distribution_major_version == "7"
-    - ansible_architecture == "ppc64le"
-  tags: nvidia_cuda_toolkit
-
 - name: Download NVidia CUDA Repo RPM - RHEL 7 on PPC64LE
   get_url:
     url: https://developer.download.nvidia.com/compute/cuda/repos/rhel7/ppc64le/cuda-repo-rhel7-9.0.176-1.ppc64le.rpm
```

Comment:
Walking through the issue to #467, I see @keithc-ca mentions v9.2.148 where this installs v9.0.176.  @keithc-ca is there an OpenJ9 support statement for the version of cuda on PPC64LE?

Reply:
We will need to update to 9.2 to get a version that is compatible with gcc 7. CUDA 9.1 `crt/host_config.h` says:
```c
#if defined(__GNUC__)
#if __GNUC__ > 6
#error -- unsupported GNU version! gcc versions later than 6 are not supported!
```
Older versions have even lower limits on the version of gcc.